src/Activity.js

```diff
@@ -7,8 +7,20 @@ class Activity {
     this.completed = false;
     this.id = Date.now();
   }
-  startTimer() {
-
+  startTimer(minutes, seconds) {
+   setInterval(function () {
+     if(seconds > 0) {
+       seconds -= 1;
+     } else if (minutes > 0) {
+       minutes -= 1;
+       seconds = 59;
+     } else {
+       return startTimerButton.innerText = "FINISHED!";
```

Comment:
We should have some math that takes into account if the user puts in more than 59 seconds.

Reply:
What is math?